### DISCUSSION TITLE
Publishing to Maven Central

### DISCUSSION THREAD

**Discussion by rotty3000** [librewsdl4j/libre-wsdl4j#10] [Q&A]
Any chance this could get published to Maven Central?

That of document a way to use the Github Package as a `<repository>` without having to authenticate?

**Comment by andreasrosdalw** [comment]


**Reply by andreasrosdalw** [reply]
libre-wsdl4j is published here now:
https://mvnrepository.com/artifact/io.github.librewsdl4j
